phpunit.xml.dist

```diff
@@ -10,10 +10,14 @@ Requires PHPUnit ^9
         </testsuite>
         <!-- Framework ORM tests are split up to run in parallel -->
         <testsuite name="framework-core">
-            <directory>tests/php</directory>
-            <exclude>
-                <directory>tests/php/ORM</directory>
-            </exclude>
+            <directory>tests/php/Control</directory>
+            <directory>tests/php/Core</directory>
+            <directory>tests/php/Dev</directory>
+            <directory>tests/php/Forms</directory>
+            <directory>tests/php/i18n</directory>
+            <directory>tests/php/Logging</directory>
+            <directory>tests/php/Security</directory>
+            <directory>tests/php/View</directory>
```

Comment:
Explicitly listing each item isn't maintainable - we will innevitably forget to add new directories here when we create them.

Reply:
then the either way is to move test files into one folder per test suite that can run in parallel

using "exclude" is not really maintanable either since it's deprecated